src/index.js

```diff
@@ -69,6 +69,28 @@ const randomCirclePoint = (centerPoint, radius, randomFn = Math.random) => {
   );
 };
 
+/**
+ *
+ * Given a center point C and a smaller radius and bigger radius, return a point that is
+ *  between the two circles
+ */
+const randomPointBetweenCircles = (
```

Comment:
Can you please change `randomPointBetweenCircles` to `randomAnnulusPoint`? [Annulus](https://en.wikipedia.org/wiki/Annulus_(mathematics)) seems like the correct math term for this.

In the comment above could pleaes add something like:

```js
/**
* Returns a random point in a region bounded by two concentric circles (annulus). 
*  ...
*/
```